.lintstagedrc.mjs

```diff
@@ -0,0 +1,6 @@
+export default {
+  '*.{js,jsx,ts,tsx,json,css,scss,sass,md,yml,yaml,cjs,mjs,cts,mts,html,htm,babelrc}':
+    'prettier --write',
```

Comment:
It would be nice to experiment a bit here and try out:
- [biome](https://biomejs.dev/guides/getting-started/) for formatting and linting
- [lefthook](https://github.com/evilmartians/lefthook) for git hooks (should be faster than lintstaged)

Speed doesn't matter much in a project of this size, but curious to see if these solutions are really production ready.

Reply:
We can definitely take a look at them and test the integration in a PR.

If there are no major issues we can definitely use them.